README.md

```diff
@@ -228,26 +224,37 @@ GTMSessionFetcher *fetcher = [fetcherService fetcherWithURL:userinfoEndpoint];
 
 ### Saving to the Keychain
 
-You can easily save `GTMAppAuthFetcherAuthorization` instances to the Keychain using
-the included `GTMAppAuthFetcherAuthorization+Keychain` category.
+You can easily save `GTMAuthSession` instances to the Keychain using the `GTMKeychainStore` class.
 
 ```objc
+// Create a GIDKeychainStore instance
+GIDKeychainStore *keychainStore =
+    [[GIDKeychainStore alloc] initWithItemName:kGTMAppAuthExampleAuthorizerKey];
+    
+NSError *error;
+
 // Save to Keychain
-[GTMAppAuthFetcherAuthorization saveAuthorization:_authorization
-                                toKeychainForName:kGTMAppAuthExampleAuthorizerKey];
+[keychainStore saveAuthSession:self.authSession error:&error];
+if (error) {
+  // Handle error
+}
 
-// Restore from Keychain
-GTMAppAuthFetcherAuthorization* authorization =
-    [GTMAppAuthFetcherAuthorization authorizationFromKeychainForName:kGTMAppAuthExampleAuthorizerKey];
+// Retrieve from Keychain
+self.authSession = [keychainStore retrieveAuthSessionAndReturnError:&error];
```

Comment:
@mdmathias should this be `retrieveAuthSessionWithError:`?

Reply:
https://github.com/google/GTMAppAuth/pull/216